package.json

```diff
@@ -19,9 +19,10 @@
         "stylelint": "stylelint \"kuma/static/styles/**/*.scss\"",
         "eslint": "eslint --ext .js --ext .jsx kuma/",
         "optim:svg": "svgo jinja2/includes/icons/* --disable={removeViewBox,removeTitle} --multipass --pretty",
-        "precommit": "npm run eslint && npm run stylelint",
+        "precommit": "npm run eslint && npm run stylelint && npm run pretty",
```

Comment:
How about changing that last word from `pretty` to `format`?
[More thoughts here](https://github.com/mozilla/kuma/pull/5526#issuecomment-506813790)